frontend_tests/node_tests/markdown.js

```diff
@@ -13,13 +13,19 @@ set_global("location", {
     origin: "http://zulip.zulipdev.com",
 });
 
-const example_realm_filters = [
-    ["#(?P<id>[0-9]{2,8})", "https://trac.example.com/ticket/%(id)s"],
-    ["ZBUG_(?P<id>[0-9]{2,8})", "https://trac2.zulip.net/ticket/%(id)s"],
-    [
-        "ZGROUP_(?P<id>[0-9]{2,8}):(?P<zone>[0-9]{1,8})",
-        "https://zone_%(zone)s.zulip.net/ticket/%(id)s",
-    ],
+const example_realm_linkifiers = [
+    {
+        pattern: "#(?P<id>[0-9]{2,8})",
+        url_format: "https://trac.example.com/ticket/%(id)s",
+    },
+    {
+        pattern: "ZBUG_(?P<id>[0-9]{2,8})",
+        url_format: "https://trac2.zulip.net/ticket/%(id)s",
+    },
+    {
+        pattern: "ZGROUP_(?P<id>[0-9]{2,8}):(?P<zone>[0-9]{1,8})",
+        url_format: "https://zone_%(zone)s.zulip.net/ticket/%(id)s",
+    },
```

Comment:
Should these examples have an ID field too?

Reply:
They aren't required for these tests -- the `id`s are used in only the settings page, but I'll add them here too for completeness.

Reply:
Done!

Reply:
I also did this treatment to a couple other instances in this file, so that these dictionaries are complete even if aren't testing all of the fields.